### PR TITLE
Fix Bug: retry scrape will lost POST requestData

### PR DESCRIPTION
**Pull request by Shinku-Chen**
when retry scrape
requestData will loss in http.NewRequest

so Seek requestData before scrape.NewRequest

```
req.ContentLength  always 0 and

if req.GetBody != nil && req.ContentLength == 0 {
			req.Body = NoBody
			req.GetBody = func() (io.ReadCloser, error) { return NoBody, nil }
		}
```

### CODE REVIEW

**Comment by Shinku-Chen** [comment]
@WGH-

**Comment by Shinku-Chen** [comment]
是个好主意
现在已经按照需求修改了代码
请帮忙检查一下 感谢
----
That's a great idea.
The code has now been modified according to the requirements.
Please help me check it out, thank you.

File: colly.go [hunk 1]
```diff
@@ -233,6 +233,8 @@ var (
 	ErrQueueFull = errors.New("Queue MaxSize reached")
 	// ErrMaxRequests is the error returned when exceeding max requests
 	ErrMaxRequests = errors.New("Max Requests limit reached")
+	// ErrRetryBodyUnSeekable is the error when retry with not seekable body
+	ErrRetryBodyUnSeekable = errors.New("Retry Body Unseekable")
```
Comment (WGH-) [inline colly.go:237]:
```suggestion
	ErrRetryBodyUnseekable = errors.New("Retry Body Unseekable")
```

Comment (WGH-) [reply colly.go:237]: And please replace UnSeeakble with Unseekable in other places

**Comment by WGH-** [comment]
Otherwise, looks good

**Review by WGH- — APPROVED** [review]

**Comment by WGH-** [comment]
And thank you!

**Pull request merged**